django_migration_vis/management/commands/visualizemigrations.py

```diff
@@ -58,7 +84,7 @@ def _add_dependencies(self, node):
             else:
                 self._add_edges(dep, self._get_tuple(node))
 
-    def render(self, save_loc):
+    def _render(self, save_loc):
```

Comment:
this is a question: for Django perspective only the `handle` method should be visible outside but in general `render` method is the method should be visible from outside or to Django interface of `handle`. Still does it make more sense to keep `render` as private?

Reply:
The idea to hide `render` is that `render` relies on `self.picture` which is only initialized in `handle`. If `render` is public there is no guarantee that `handle` has been called before. And moving `self.picture` initialization to `__init__` is not trivial (but may work).

Reply:
agreed